sycl/plugins/esimd_emulator/pi_esimd_emulator.cpp

```diff
@@ -66,7 +66,7 @@ pi_result getInfoImpl(size_t ParamValueSize, void *ParamValue,
 template <typename T>
 pi_result getInfo(size_t ParamValueSize, void *ParamValue,
                   size_t *ParamValueSizeRet, T Value) {
-  auto assignment = [](void *ParamValue, T Value, size_t ValueSize) {
+  auto assignment = [](void *ParamValue, T Value, size_t) {
```

Comment:
I don't remember any agreement on this. So, this may be Ok.
In those cases where the type does not describe the parameter it is ok to omit the name, on other like 'size_t' it may be still good to have a name in comment section. For example:
```suggestion
  auto assignment = [](void *ParamValue, T Value, size_t /*ValueSize*/) {
```

Reply:
In other plug-in implementations, `(void)XXX;` is used. I introduced `ARG_UNUSED` macro for suppressing warning messages from unused arguments.